tldr/api.py

```
@@ -1568,6 +1618,16 @@ def get_code_structure(
 
     result = {"root": str(root), "language": language, "files": []}
 
+    # Handle single-file input: rglob("*") on a file returns empty iterator
+    if root.is_file():
+        if root.suffix in extensions:
+            try:
+                info = _extract_file_impl(str(root))
+                result["files"].append(_build_file_entry(info.to_dict(), root.name))
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)